wc.go

```diff
@@ -33,12 +32,13 @@ func (s Stream) Wc() Count {
 		count.Words += countWords(scanner.Text())
 	}
 	if err := scanner.Err(); err != nil {
-		errors = multierror.Append(errors, fmt.Errorf("scanning stream: %s", err))
+		merr = errors.Join(merr, fmt.Errorf("scanning stream: %w", err))
 	}
 
 	count.Stream = Stream{
 		stage: "wc",
 		r:     strings.NewReader(count.String()),
+		err:   merr,
```

Comment:
I think this might have been a bug - the error was never fed back to the `Stream` return.

Reply:
Interesting. I wonder if we can add a test for that.

Reply:
Done